pkg/tsdb/cloudwatch/metric_find_query.go

```diff
@@ -282,3 +284,38 @@ func (e *cloudWatchExecutor) resourceGroupsGetResources(ctx context.Context, plu
 
 	return &resp, nil
 }
+
+// legacy route, will be removed once GovCloud supports Cross Account Observability
+func (e *cloudWatchExecutor) handleGetLogGroups(ctx context.Context, pluginCtx backend.PluginContext, parameters url.Values) ([]suggestData, error) {
+	region := parameters.Get("region")
+	limit := parameters.Get("limit")
+	logGroupNamePrefix := parameters.Get("logGroupNamePrefix")
+
+	logsClient, err := e.getCWLogsClient(ctx, pluginCtx, region)
+	if err != nil {
+		return nil, err
+	}
+
+	logGroupLimit := defaultLogGroupLimit
+	intLimit, err := strconv.ParseInt(limit, 10, 64)
+	if err == nil && intLimit > 0 {
+		logGroupLimit = intLimit
+	}
+
+	var response *cloudwatchlogs.DescribeLogGroupsOutput = nil
```

Comment:
is there a reason this is a `var` on this line instead of it being on line 310?

```go
response, err := logsClient.DescribeLogGroups(input)
```

Reply:
no reason I can think of! I just copy and 🍝 from what we used to have but I like the suggestion! Changing now :)